docs/antrea-network-policy.md

```diff
@@ -902,25 +904,41 @@ an `appliedTo` must resolve to. More information on ClusterGroups can be found i
 **serviceAccount**: This selects all the Pods which have been assigned a specific ServiceAccount.
 For more information on its usage, refer to [this section](#serviceaccount-based-selection).
 
-**ipBlock**: This selects particular IP CIDR ranges to allow as `ingress`
-"sources" or `egress` "destinations". These should be cluster-external IPs,
-since Pod IPs are ephemeral and unpredictable.
+**ipBlock**: This selects particular IP CIDR ranges to allow as `ingress` "sources"
+or `egress` "destinations". These should be cluster-external IPs, since Pod IPs are
+ephemeral and unpredictable.
+
+Starting with Antrea v2.2, Antrea-native policies now support an `except` field under
+`ipBlock`, which can be used to exclude specific CIDRs in the base IP range:
+
+```yaml
+ipBlock:
+  cidr: 10.168.1.0/16
+  except: 
+    - 10.168.128.0/24
+```
 
-**fqdn**: This selector is applicable only to the `to` section in an `egress` block. It is used to
-select Fully Qualified Domain Names (FQDNs), specified either by exact name or wildcard
+Note that at the time of policy creation, the CIDRs in the `except` list will be validated
+to be strictly within the base CIDR range. Policy creation will fail otherwise.
+Policy rules that select ipBlocks with `except` fields will simply not match traffic whose
+addresses are included in the `except` fields. Those packets are subject to further policy
+evaluations for lower priority rules. A large number of policy rules with `except` ipBlocks
+can also have negative impacts on Antrea memory usage.
```

Comment:
It's fine by me. @tnqn do you know if a high number of flows can also impact latency / throughput. I imagine that it is possible with a large number of diverse connections.

Reply:
>  An IPBlock with an except ipBlock can result in 31 IPNets in the worst case

technically, I can have multiple except cidrs and generate many more IPNets than that

but yes, I agree with the overall statement